crd-docs/cr/kiali.io_v1alpha1_kiali.yaml

```diff
@@ -409,8 +409,13 @@ spec:
           http: "requests"
           tcp: "sent"
       i18n:
-        default: "en"
-        languages: ["en"]
+        language: "en"
+        show_selector: true
+      list:
```

Comment:
what is "list" ??

Reply:
As for the original question, the `list` section can be added, it is a documented feature.